packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -302,6 +302,15 @@ class DependencyExtractionWebpackPlugin {
 				assetData.type = 'module';
 			}
 
+			if ( compilation.options?.optimization?.runtimeChunk !== false ) {
```

Comment:
Should it also be different than `undefined`?

Reply:
The default is `false` as explained:

https://webpack.js.org/configuration/optimization/#optimizationruntimechunk

Other allowed values are `true`, any string and an object.